packages/connect-common/files/firmware/t2t1/releases.json

```diff
@@ -228,138 +228,5 @@
         "fingerprint_bitcoinonly": "bddc0fd3b52fd32d94b776048f62b3d03dcb6ab90140e482a042a2863093115f",
         "notes": "https://blog.trezor.io/details-of-firmware-updates-for-trezor-one-version-1-9-0-and-trezor-model-t-version-2-3-0-46deb141fc09",
         "changelog": "* Introduce Wipe code\n* Introduce SD card protection\n* Introduce passphrase cache\n* U2F UX improvements\n* Security fixes"
-    },
```

Comment:
till now, the consensus was only to set a newer firmware to `required:true`. Lets stick to it. I think the old records are needed for firmware update... maybe..